docs/tips-and-tricks.md

```diff
@@ -120,3 +158,17 @@ img.twitter-icon {
   background: url("data:image/svg+xml,@{svg}") no-repeat !important;
 }
 ```
+&nbsp;
+
+### How to get raw RGB values from a color
+
+You can use the following snippet to get the raw RGB values from a color:
+
+```less
+#rgbify(@color) {
+  @rgb-raw: red(@color), green(@color), blue(@color);
+}
+
+// Below is an example of how to use the above snippet, where @rosewater is a color variable
+#rgbify(@rosewater)[];
```

Comment:
```suggestion
--ctp-base: #rgbify(@base) []; // -> 30, 30, 46
```

These are the actual color values for mocha, I don't know if we need to mention what flavor it is but I think it is helpful to understand what the output looks like.